packages/common/src/types/schema.ts

```diff
@@ -915,6 +881,13 @@ export type InvoiceQuery = {
         entryDatetime: any;
         finalisedDatetime?: any | null | undefined;
         invoiceNumber: number;
+        draftDatetime?: any | null | undefined;
+        allocatedDatetime?: any | null | undefined;
+        pickedDatetime?: any | null | undefined;
+        shippedDatetime?: any | null | undefined;
+        deliveredDatetime?: any | null | undefined;
```

Comment:
this is whats generated which doesn't seem ideal 🤔

Reply:
Ah, IC, the custom scalars `DateTime` and `NaiveDate` are of type `any`. I'm sure there must be a way to fix that.